sdk/streamlib-python-wheel/python/streamlib/__init__.py

```
@@ -28,22 +28,30 @@
 from ._engine import Runtime as _NativeRuntime
 from ._engine import RuntimeContextFullAccess as RuntimeContextFullAccess
 from ._engine import RuntimeContextLimitedAccess as RuntimeContextLimitedAccess
+from ._engine import TestPatternSource as TestPatternSource
 from ._engine import media_clock_now_ns as media_clock_now_ns
 from ._engine import monotonic_now_ns as monotonic_now_ns
 from ._processor_declaration import input as input
```

Ruff (0.16.1)

[error] Import `input` is shadowing a Python builtin

(A004)